dev/dune

```diff
@@ -10,7 +10,9 @@
 (rule
   (targets dune-dbg)
   (deps dune-dbg.in
+        %{project_root}/theories/Init/Prelude.vo
```

Comment:
I don't think that's a good idea, sometimes I want to debug why my change broke an init file.

Reply:
OK